dpbench/benchmarks/knn/knn_numba_npr.py

```diff
@@ -84,7 +84,7 @@ def knn(
                     queue_neighbors[index, 0] = new_distance
                     queue_neighbors[index, 1] = new_neighbor_label
 
-        v_to_c_i = votes_to_classes[i]
+        v_to_c_i = np.copy(votes_to_classes[i])
```

Comment:
Not necessary for numba_npr

Reply:
This is need to avoid divided by zero: 
/localdisk/work/mingjie2/sandbox/review/dpbench/dpbench/infrastructure/utilities.py:34: RuntimeWarning: divide by zero encountered in double_scalars
  return np.linalg.norm(ref - val) / np.linalg.norm(ref)
Relative error: inf

Reply:
Then add a comment?